src/EkoPlayer.js

```diff
@@ -1,7 +1,11 @@
-import deepmerge from 'deepmerge';
-import EventEmitter from 'eventemitter3';
+/* eslint-disable new-cap */
+import ekoEmbedFactory from './lib/ekoEmbed/factory';
+import coverFactory from './lib/cover/coverFactory';
```

Comment:
Might be a bit nit picky, so feel free to ignore, but we should consider changing the filename for consistency (i.e. `ekoEmbed/factory` vs. `cover/coverFactory`).